src/rust/generic-subgraph-generator/src/models/process/port_bind.rs

```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Hash, Serialize, Deserialize)]
+pub struct ProcessPortBindLog {
```

Comment:
This was previously unimplemented. I've decided to leave it as such as it's not exactly in the scope of this diff to add functionality.

Reply:
Might be worth adding an issue to track this?

Reply:
Done :D https://github.com/grapl-security/grapl/issues/307

Reply:
yeah +1